### PR TITLE
Remove some old code artifacts and two insecure hash methods

### PR DESCRIPTION
**Pull request by dkanada**
I also couldn't build without changing the import statement for the LruCache class.

### CODE REVIEW

**Comment by dkanada** [comment]
I still think using sha256 or something as a replacement isn't a terrible idea but these two methods are outdated regardless.

File: library/src/main/java/org/jellyfin/apiclient/interaction/websocket/ApiWebSocket.java [hunk 1]
```diff
@@ -290,63 +287,30 @@ private void OnGeneralCommand(String json)
...
-                tangible.RefObject<Long> tempRef_expected = new tangible.RefObject<Long>(expected);
-                LongHelper.TryParseCultureInvariant(timeoutMs, tempRef_expected);
-                expected = tempRef_expected.argValue;
-
                 MessageCommand command = new MessageCommand();
-
                 command.setHeader(header);
                 command.setText(text);
-                command.setTimeoutMs(expected);
+                command.setTimeoutMs(Long.parseLong(timeoutMs));
```
Comment (thornbill) [inline library/src/main/java/org/jellyfin/apiclient/interaction/websocket/ApiWebSocket.java]: We should probably catch `NumberFormatException`s here like the `TryParseCultureInvariant` method did.

File: library/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java [hunk 2]
```diff
@@ -473,16 +473,15 @@ public DeviceProfile()
 
 	public final TranscodingProfile GetAudioTranscodingProfile(String container, String audioCodec)
 	{
-		container = StringHelper.TrimStart((container != null) ? container : "", '.');
-
+		container = (container != null) ? container.replace(".", "") : "";
```
Comment (thornbill) [inline library/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java]: This no longer removes leading whitespace. May introduce new issues?

File: library/src/main/java/org/jellyfin/apiclient/interaction/connectionmanager/ConnectionManager.java [hunk 3]
```diff
@@ -690,7 +689,7 @@ String NormalizeAddress(String address) throws IllegalArgumentException {
             throw new IllegalArgumentException("address");
         }
 
-        if (StringHelper.IndexOfIgnoreCase(address, "http") == -1)
+        if (!address.contains("http"))
```
[the remark this answers is not in the document]
Comment (dkanada) [reply library/src/main/java/org/jellyfin/apiclient/interaction/connectionmanager/ConnectionManager.java]: I used patterns for all these cases but I don't plan on actually fixing the root issue in this pull request - might fix it eventually.

File: library/src/main/java/org/jellyfin/apiclient/interaction/websocket/ApiWebSocket.java (hunk 1 above)
Comment (dkanada) [reply library/src/main/java/org/jellyfin/apiclient/interaction/websocket/ApiWebSocket.java]: I think the best short term plan is to pass the exception up a level so the app can handle the error with its chosen logging implementation.

File: library/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java (hunk 2 above)
Comment (dkanada) [reply library/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java]: I honestly don't think this one will be an issue, do you think so?

**Pull request merged**